server/views/partials/audienceTag.njk

```diff
@@ -0,0 +1,7 @@
+{% from "govuk/components/tag/macro.njk" import govukTag %}
+
+{% macro audienceTag(tag) %}
+  <p class="govuk-body">
```

Comment:
I forget that we don't need the `govuk-body` class as we have `$govuk-global-styles: true;` set.  I know this is still used in a few other places and won't break anything so happy to leave and they can be removed as and when we make changes in the future.

Reply:
Oh nice - didn't know about that. It looks like `govuk-link` might also be unnecessary: https://github.com/alphagov/govuk-frontend/blob/8eeca8e8a7980dfaceb6619ca97084fdb63634f6/packages/govuk-frontend/src/govuk/core/_global-styles.scss#L4-L12

I'll leave it for now though and add a card for it, as some of the integration test selectors might need updating